holidays/countries/south_georgia_and_the_south_sandwich_islands.py

```
@@ -0,0 +1,152 @@
+
+        self._add_observed(
+            # Boxing Day.
+            self._add_christmas_day_two(tr("Boxing Day")),
+            rule=SAT_SUN_TO_NEXT_MON_TUE + MON_TO_NEXT_TUE,
+        )
+
+        # Christmas Day.
+        self._add_observed(self._add_christmas_day(tr("Christmas Day")))
+
+
+class GS(SouthGeorgiaandtheSouthSandwichIslands):
```

Ruff (0.12.2)

Undefined name `SouthGeorgiaandtheSouthSandwichIslands`

(F821)